api/v1alpha4/azurecluster_conversion.go

```diff
@@ -42,6 +42,54 @@ func (src *AzureCluster) ConvertTo(dstRaw conversion.Hub) error {
 	// Restore list of virtual network peerings
 	dst.Spec.NetworkSpec.Vnet.Peerings = restored.Spec.NetworkSpec.Vnet.Peerings
 
+	// Restore API Server LB IP tags.
+	for _, restoredFrontendIP := range restored.Spec.NetworkSpec.APIServerLB.FrontendIPs {
+		for i, dstFrontendIP := range dst.Spec.NetworkSpec.APIServerLB.FrontendIPs {
+			if restoredFrontendIP.Name == dstFrontendIP.Name && restoredFrontendIP.PublicIP != nil {
+				dstFrontendIP.PublicIP.IPTags = restored.Spec.NetworkSpec.APIServerLB.FrontendIPs[i].PublicIP.IPTags
```

Comment:
ditto my above comment for all of these patterns (we can discuss on the thread above)